app/assets/src/components/common/Header.jsx

```diff
@@ -160,10 +160,6 @@ class Header extends React.Component {
             // Initialize the toast container - can be done anywhere (has absolute positioning)
           }
           <ToastContainer />
-          <iframe
```

Comment:
Not sure if this would cause an error, but better to not need the background_refresh